blogposts/2021/release-3.27.md

```diff
@@ -27,6 +27,18 @@ changelogItems:
   - description: "New changes of a Perforce depot will now be reflected in `master` branch after the initial clone."
     url: https://github.com/sourcegraph/sourcegraph/pull/19718
     category: Admin
+  - description: "Search functionality via built-in `contains` predicate: `repo:contains(...)`, `repo:contains.file(...)`, `repo:contains.content(...)`, repo:contains.commit.after(...)`."
+   url: https://github.com/sourcegraph/sourcegraph/issues/18584
+   category: Search
+  - description: "Database encryption, external service config \u0026 user auth data can now be encrypted in the database using the `encryption.keys` config. See the docs for more info."
```

Comment:
@christinaforney - recommendation: User auth data can now be encrypted in the database using the `encryption.keys` config. 

Category: Admin

Reply:
```suggestion
  - description: "User auth data can now be encrypted in the database using the `encryption.keys` config."
```